os/kernel/log_dump/log_dump.c

```diff
@@ -505,6 +505,26 @@ int log_dump_save(char ch)
 	return LOG_DUMP_OK;
 }
 
+size_t log_dump_get_size(void)
+{
+	size_t size = 0;
+	struct log_dump_chunk_s* head = (struct log_dump_chunk_s *)sq_peek(&log_dump_chunks);
```

Comment:
In code readability, head != tail? It looks strange.
How about renaming the variable like log_node?
It does not have the head always.